samples/data-protection-toolkit-minimal.yaml

```diff
@@ -71,11 +71,10 @@ projects:
                     - containerPort: 2575
                       protocol: TCP
                       name: "port"
-                  command:
-                    - "/import/bin/import"
+                  args:
                     - "--dimse_aet=IMPORTADAPTER"
                     - "--dimse_port=2575"
-                    - "--dicomweb_address=https://healthcare.googleapis.com/v1beta1/projects/my-project/locations/us-central1-c/datasets/dataset/dicomStores/dicomStores/dicomWeb"
+                    - "--dicomweb_address=https://healthcare.googleapis.com/v1/projects/my-project/locations/us-central1-c/datasets/dataset/dicomStores/dicomStores/dicomWeb"
```

Comment:
We need to update the container image to 0.2.1 as well if we are switching to args.

Reply:
Ok